.github/workflows/test-push-live-k8s-cluster.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         uses: helm/kind-action@v1.2.0
 
       - name: Run live-k8s tests
-        run: python tests/run_live_k8s_tests.py -i cmdline_live_k8s_tests.yaml
+        run: python tests/run_all_tests.py --type=k8s_live_general
```

Comment:
I think we can merge the `test-push` and the `test-push-live-k8s-cluster` workflows into one work flow.

Reply:
done